src/ui/components/DashKit/plugins/Control/Control.tsx

```diff
@@ -987,19 +931,12 @@ class Control extends React.PureComponent<PluginControlProps, PluginControlState
             key: fieldId,
             value: preparedValue as string,
             editMode,
-            onChange: (value: string | string[]) => {
-                const valueWithOperation = addOperationForValue({
-                    value,
-                    operation: source.operation,
-                });
-
-                this.onChange(fieldId, valueWithOperation);
-            },
+            onChange,
             onOpenChange: this.onOpenChange,
             loadingItems,
-            placeholder: Utils.isEnabledFeature(Feature.EmptySelector)
-                ? i18n('placeholder_empty')
-                : undefined,
+            placeholder,
+            isValueRequired: source.isValueRequired,
+            isValidationError: Boolean(validationError),
```

Comment:
If it's bool it should be and `hasValidationError` is not the flag that is telling user that this field is for validation errors output. Than i suggest using `invalid` or `isValid` as name would be better.